src/openads/infrastructure/position/positionSegmentationChangedObserver.js

```diff
@@ -0,0 +1,10 @@
+const positionSegmentationChangedObserverFactory = appnexusConnector => ({payload, dispatcher}) =>
+  appnexusConnector
+    .modifyTag(payload.id, {
```

Comment:
Not sure about this one. 
If I am only updating the segmentation, this line will put the sizes to undefined

Reply:
ups, totally agree, let me fix it, sorry ;)

Reply:
Ok sorry I forgotten that the observer recieve on the payload the diff already calculate on Domain, so any field not updated will have the previous value, so you will never set an undefined because you didn't updated that field ;)

Reply:
OK MAN

![](https://media.giphy.com/media/3ohc12h6fY2NgArgWc/giphy.gif)